modules/ROOT/pages/cloudhub-impaired-worker.adoc

```diff
@@ -1,17 +1,59 @@
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-fabric/deploy-to-runtime-fabric", title="Runtime Fabric"]
 
-When a worker is impaired, CloudHub automatically attempts to restart the worker and relaunch the application. If a retry is successful, the underlying worker and the Mule application are recovered without requiring any further action.
+CloudHub continuously performs health checks on the underlying virtual machines where Mule applications are running. 
+When a worker is impaired, CloudHub automatically attempts to restart the worker and relaunch applications.
 
-[NOTE]
+A worker is impaired when there is a hardware or software failure that negatively impacts the running application deployed on the worker.
+Hardware and software failures include network unavailability, memory issues, server hardware failures, and file corruption.
+
+When CloudHub restarts the impaired worker:
+
+* If a retry is successful, the underlying worker and app restart without any further action.
++
 It might take up to five minutes for a worker to restart.
+* If the application fails to start after five attempts, the status of the application is *Failed* or *Unrecoverable Runtime Error*.
++
+In this case, you must manually restart the application.
+
+The logs indicate when a worker is restarted automatically and any failures.
 
-If the application fails to start after five attempts, the status of the application is *Failed* or *Unrecoverable Runtime Error*. In this case, you must manually redeploy and restart the application.
+[NOTE]
+The automatic worker-restart functionality is different from the automatic application-restart functionality, which is configurable using the *Automatically restart application when not responding* option on the *Settings* page for the app. 
+CloudHub automatically attempts to restart impaired workers, regardless of the app setting. 
 
 == Scheduled Maintenance Activities
 
-CloudHub checks for scheduled maintenance activities on the worker instances running the Mule applications. The following maintenance activities can cause a potential downtime for the running applications: reboot, stop, start, or retirement of the instances. To avoid downtime, Cloudhub checks for these scheduled events and sends you a notification in advance to restart your application with zero downtime.
+CloudHub checks for scheduled maintenance activities for worker instances that cause potential downtime for the running applications: reboot, stop, start, or retirement of the instances.
```

Comment:
transition and chrono reorg suggestion:

"In addition to automatically restarting failed workers, CloudHub also restarts workers after scheduled maintenance. One week before such maintenance, an icon appears <here>. Clicking the icon displays a notification that tells you when can expect Cloudhub to restart the worker and your apps."

QUESTIONS:
Does the worker and app continue to work during the maint?
Do they stop and then restart, or just restart?
Does the icon appear in the top right, or the notification; they are not the same thing, right?
Why does the user need to know about the restart: is there an up-to-5-min lull?
Why would the user want to manually restart: to avoid the lull, or only if the app is not configured to restart or fails?
Should we combine the texts about failure and maint, if the chronology is the same?
Does the message appear in both cases: failure and maint?
If a worker is retired, is this transparent: is there a replacement?

Reply:
No.
Restart.
The screenshot shows the icon. You click the icon to see the notification.
I guess so.
To schedule at their convenience.
No.
Only maint.
Yes.